src/genie/libs/parser/iosxe/tests/c9300/test_show_platform.py

```diff
@@ -155,6 +156,717 @@ def test_show_inventory_c9300(self):
+ Mac Address Table      EM           I       32768       33    0.10%        0        0        0       33
+ Mac Address Table      TCAM         I        1024       21    2.05%        0        0        0       21
+ L3 Multicast           EM           I        8192        0    0.00%        0        0        0        0
+ L3 Multicast           TCAM         I         512       67   13.09%        3       64        0        0
+ L2 Multicast           EM           I        8192        0    0.00%        0        0        0        0
+ L2 Multicast           TCAM         I         512       11    2.15%        3        8        0        0
+ IP Route Table         EM           I       24576       40    0.16%       25        4       11        0
+ IP Route Table         TCAM         I        8192       76    0.93%       29       44        2        1
+ QOS ACL                TCAM         IO       5120       81    1.58%       27       36        0       18
+ Security ACL           TCAM         IO       5120      129    2.52%       26       58        0       45
+ Netflow ACL            TCAM         I         256        7    2.73%        3        2        0        2
+ PBR ACL                TCAM         I        1024       22    2.15%       16        6        0        0
+ Netflow ACL            TCAM         O         768        7    0.91%        3        2        0        2
+ Flow SPAN ACL          TCAM         IO       1024       13    1.27%        3        6        0        4
+ Control Plane          TCAM         I         512      263   51.37%      114      106        0       43
+ Tunnel Termination     TCAM         I         512       51    9.96%       41       10        0        0
+ Lisp Inst Mapping      TCAM         I        2048        1    0.05%        0        0        0        1
+ Security Association   TCAM         I         256        3    1.17%        1        2        0        0
+ CTS Cell Matrix/VPN
+ Label                  EM           O        8192        0    0.00%        0        0        0        0
+ CTS Cell Matrix/VPN
+ Label                  TCAM         O         512        1    0.20%        0        0        0        1
+ Client Table           EM           I        4096       11    0.27%        0        0        0       11
+ Client Table           TCAM         I         256        0    0.00%        0        0        0        0
+ Input Group LE         TCAM         I        1024        0    0.00%        0        0        0        0
+ Output Group LE        TCAM         O        1024        0    0.00%        0        0        0        0
+ Macsec SPD             TCAM         I         256        2    0.78%        0        0        0        2
+    '''}
+
+    golden_parsed_output = {
+          '0': {
+            'Mac Address Table': {
+              'EM': {
+                'I': {
+                  'max_entry': '32768',
+                  'use_entry': '33',
+                  'percent': '0.10',

```

Comment:
change to 'used_percent' to make it clear it's for 'used' entries.

Reply:
not updated yet?

Reply:
I have updated percent to used_percent. Please check the latest code.

Reply:
confirmed